.github/workflows/CI.yml

```diff
@@ -0,0 +1,95 @@
+on:
+  push:
+    branches: [master]
+  pull_request:
+
+name: Continuous integration
+
+jobs:
+  check:
+    name: Check
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        rust:
+          - stable
+          - nightly
+          - 1.39.0 # MSRV
```

Comment:
What's a meaningful value to use here? I basically want "oldest rust I can right now" but still 2018 for those import things (right?)

Reply:
you can tweak this number lower and lower until you find a number that doesn't build your project. Then you'll *know* what the MSRV is for your project